src/webgpu/api/validation/image_copy/buffer_texture_copies.spec.ts

```diff
@@ -304,3 +306,46 @@ g.test('depth_stencil_format,copy_buffer_offset')
       unreachable();
     }
   });
+
+g.test('texture_buffer_usages')
+  .desc(
+    `
+  Tests calling copyTextureToBuffer or copyBufferToTexture with the texture and the buffer missed
+  COPY_SRC, COPY_DST usage respectively.
+    - texture and buffer {with, without} COPY_SRC and COPY_DST usage.
+  `
+  )
+  .paramsSubcasesOnly(u =>
+    u //
+      .combine('textureUsage', kTextureUsages)
+      .combine('bufferUsage', kBufferUsages)
+      .combine('copyType', ['CopyB2T', 'CopyT2B'] as const)
```

Comment:
nit: I would make copyType a non-subcase parameter (move it up to the case level)

Reply:
Done.